selftest/uprobe/go.sum

```diff
@@ -0,0 +1,16 @@
+github.com/aquasecurity/libbpfgo v0.1.0 h1:14V7ISF45XVqwBre+7NCDByy77OuwFgzDjVR+0JltZ8=
+github.com/aquasecurity/libbpfgo v0.1.0/go.mod h1:/+clceXE103FaXvVTIY2HAkQjxNtkra4DRWvZYr2SKw=
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
are these needed? maybe you forgot a `go mod tidy`?

Reply:
I'm not sure why this is, but I did run a go mod tidy (which made changes) but this is still there. It's imported by testify

Reply:
ah yeah, probably testify brought it in